misc/testerina/modules/testerina-core/src/main/ballerina/execute.bal

```diff
@@ -293,13 +293,10 @@ function skipDataDrivenTest(TestFunction testFunction, string suffix, TestType t
     if (!suffixMatch) {
```

Comment:
```suggestion
    if !suffixMatch {
```

Better readability if we can remove the parentheses from the if conditions. If I recall, this is the practice that we have used from the beginning of the Testerina implementation. Address for other applicable statements.